tests/cephfs/cephfs_top/cephfs_top_initiate.py

```diff
@@ -0,0 +1,162 @@
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+Testing description:
+
+Verify cephfs-top initial status and basic configuration
+
+Steps to Reproduce:
+1. Try to test install cephfs-top package
+2. Check if cephfs-top packages install is working well
+3. Check if cephfs-top is working without enabling mgr stats
+4. check if cephfs-top client is created well with appropriate permission
+5. check if cephfs-top -h,--help is working well
+"""
+
+
+def run(ceph_cluster, **kw):
```

Comment:
@julpark-rh This step seems missing in automation "Create a client for non-default user, run cephfs-top --id <client-name>" ,Can it be implemented?